contracts/NameWrapper.sol

```diff
@@ -544,7 +545,7 @@ contract NameWrapper is
             "NameWrapper: Token id does match keccak(label) of label provided in data field"
         );
 
-        bytes32 labelhash = keccak256(bytes(label));
+        bytes32 labelhash = bytes32(tokenId);
```

Comment:
probably better to set labelhash before require so that you don't have to call `bytes32(tokenId)` twice.
```
        bytes32 labelhash = bytes32(tokenId);
        require(
            keccak256(bytes(label)) == labelhash,
            "NameWrapper: Token id does match keccak(label) of label provided in data field"
        );
```